runtime/compiler/p/codegen/ForceRecompilationSnippet.cpp

```diff
@@ -100,22 +105,51 @@ uint8_t *TR::PPCForceRecompilationSnippet::emitSnippetBody()
       startPCReg->setRegisterFieldRS((uint32_t *)buffer);
       *(uint32_t *)buffer |= llval;
       buffer += PPC_INSTRUCTION_LENGTH;
+
+      TR::Relocation *startPCRelo = new (cg()->trHeapMemory()) TR::ExternalRelocation(firstInstruction,
+                                                                                     (uint8_t *)startPC,
```

Comment:
Might be better to just pass in `NULL` here; from the code in `p/codegen/J9AheadOfTimeCompile.cpp` it doesn't look like `targetAddress` is used, so by passing in NULL we reduce misleading a casual read of this code.

Reply:
Yes, that would be better. I think adding this record needs to be guarded by `needRelocationsForBodyInfoData` as well.